pkg/generate/app/cmd/newapp.go

```diff
@@ -486,6 +490,17 @@ func (c *AppConfig) search(components app.ComponentReferences) error {
 	return errors.NewAggregate(errs)
 }
 
+func (c *AppConfig) detectPartialMatches(components app.ComponentReferences) error {
```

Comment:
What scenario would users get this?  I would expect this to be resolve resulting in partial matching